python/master.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul 25 11:28:01 2019
+
+@author: marcdimeo
+"""
+
+import mygene
+import pandas as pd
+import numpy as np
+from sklearn import preprocessing
+import scipy.io as spio
+import os
+from scipy.stats.stats import pearsonr
+import matplotlib.pyplot as plt
+import seaborn as sns
+import plotly.io as pio
+import plotly.figure_factory as ff
+from scipy.spatial.distance import pdist, squareform
+#import plotly.graph_objects as go
+
+"""
+FUNCTIONS
```

Comment:
Make sure your code follows this style guide: https://realpython.com/python-pep8/